ckan/views/group.py

```diff
@@ -302,6 +302,9 @@ def pager_url(q: Any = None, page: Optional[int] = None):
 
     extra_vars["facet_titles"] = facets
 
+    # Sanitize the query by removing double quotes (if any) to avoid solr issues
+    q = q.replace('"', '') if '"' in q else q
```

Comment:
It's not a valid fix, since only unclosed quotes cause an error and only when they are not enclosed into a single quotes, while paired quotes may be used to search literal values.